scss/layout/sidebar.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../../stanford_basic/scss/components/sidebar/sidebar";
+@import "config/config";
+//// Decanter config.
+@import 'decanter/core/utilities/decanter-utilities';
+// Grab our mixins.
+@import "utilities/utilities";
+
+body.with-banner {
+  header {
+    //min-height: 14px;
+
+    #block-matson-branding {
+      position: absolute;
+      padding: 20px;
+      left: 128px;
```

Comment:
This is tricky because if you look at the mockups, the banner photos are supposed to go all the way to the left edge, but the text should all get moved over so it's not under the sidebar.  This is a very unusual sidebar layout.

I know having a sidebar at all runs against Stanford design preferences, and we already have another design for subnav on this site: https://projects.invisionapp.com/d/main#/console/10787452/231823690/preview  Considering how tricky the sidebar is to implement, would you consider removing it entirely and using the other subnav design on department pages?